tsl/test/sql/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ if (${PG_VERSION_MAJOR} GREATER "9")
     compression_hypertable.sql
     compression_segment_meta.sql
     compression_bgw.sql
-    compress_bgw_drop_chunks.sql
+    compress_bgw_reorder_drop_chunks.sql
```

Comment:
The added test doesn't fit well into this test file, since the file is about compression background worker, while the test is about reorder background worker.

Reply:
Will it be more natural to add the test to `bgw_reorder_drop_chunks.sql` and rename it to be for compressed and drop chunks?

Reply:
But `bgw_reorder_drop_chunks.sql` runs for all versions of PG while `compress_bgw_reorder_drop_chunks.sql` runs only for version higher than 9. If I add compression tests to `bgw_reorder_drop_chunks.sql` it will fail for PG 9.6. So I added them to `compress_bgw_reorder_drop_chunks.sql`.

Reply:
You are right.

Reply:
I can think to improve the name of this test: `bgw_compress_reorder_drop_chunks.sql`. However, I don't know how others will like this idea :)

Reply:
No problem, if you prefer another name or a separated file for this test - let me know.